script.js

```diff
@@ -0,0 +1,298 @@
+localStorage.setItem('caps', 'smallLetter');
+localStorage.setItem('lang', 'Eng');
+let lang = localStorage.getItem('lang');
+if (sessionStorage.hasOwnProperty('lang')) {
+    lang = sessionStorage.getItem('lang');
+}
+const body = document.querySelector('body');
```

Comment:
Usually `body` can be called as `document.body`. It's not nesessary to find `body` in DOM separately.